apps/judicial-system/api/src/app/modules/case/interceptors/case.transformer.spec.ts

```diff
@@ -1,6 +1,7 @@
-import { SessionArrangements } from '@island.is/judicial-system/types'
 import each from 'jest-each'
 
+import { SessionArrangements } from '@island.is/judicial-system/types'
```

Comment:
Is this an accidental import? Are you using it anywhere?

Reply:
It is used in the tests below in an each-clause.